tests/Api/Admin/ProductImagesTest.php

```diff
@@ -36,49 +38,262 @@ public function it_denies_access_to_a_product_images_list_for_not_authenticated_
     /** @test */
     public function it_gets_all_product_images(): void
     {
-        $this->loadFixturesFromFiles(['product/product_image.yaml', 'authentication/api_administrator.yaml']);
+        $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'product/product_image.yaml']);
         $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
```

Comment:
Would be cool to make use of `HeadersBuilder` https://github.com/Sylius/Sylius/blob/1.13/tests/Api/Utils/HeadersBuilder.php here and in other places in this test, since it was introduces last week.